revision in 'General Information'

TristanCode0327 opened the pull request:
1. change of 'general information" > 'Beyond the-
'classroom'
2. New webpages(Zoo Project, Student-Run Clubs, Student Competitions)
under 'Beyond the classroom'

TristanCode0327 committed:
more pages for 'general information'

nurikimchi commented:
Dropdowns under "Beyond the Classroom" go off-screen through the viewport's bottom edge. Create a new dropdown link called "Pathway Information"--similar to the existing "Beyond the Classroom" dropdown menu--as a placeholder for now. Move "Updated Classroom" and "Pathway Outcomes" into this new "Pathway Information" menu.

<img width="1280" alt="Screenshot 2023-12-31 at 10 34 26 PM" src="https://github.com/estebangarcia21/csfhs/assets/64626132/c480407c-9579-475c-96f7-74d8cb65340f">
<img width="1280" alt="Screenshot 2023-12-31 at 10 34 19 PM" src="https://github.com/estebangarcia21/csfhs/assets/64626132/622bfa09-78e1-4044-aafe-eb5d8b77250a">

TristanCode0327 committed:
new 'Pathway Information' dropdown on the navbar

TristanCode0327 committed:
new 'Pathway Information' dropdown on the navbar

nurikimchi committed:
Implement deployment fixes

nurikimchi committed:
Rough draft first styling

nurikimchi committed:
Add images

nurikimchi committed:
Update course data, add more info.

nurikimchi committed:
Add testimonials for homepage

nurikimchi committed:
Update footer to more minimalist design

Add each course name as link

nurikimchi committed:
Made classes dropdown one column, made general info into grid

nurikimchi committed:
Update title to full version of abbrev.

nurikimchi committed:
Update course homepage, add single-quote testimonial, alumni institutions, FAQ

nurikimchi committed:
Edit packages

nurikimchi committed:
Attempt at fixing broken images in Vercel deployment

nurikimchi committed:
Fix alumni institution images

nurikimchi committed:
Merge pull request #28 from nurikimchi/nuri

Updates for 25-26 course selection

nurikimchi committed:
Packages before updates

nurikimchi committed:
Update extracurricular pages

nurikimchi committed:
Add Google Form registration link

nurikimchi committed:
Update vite packages

nurikimchi committed:
Finalize renovation

nurikimchi committed:
Merge pull request #29 from nurikimchi/nuri

Updates to extracurricular opportunities; finalized upgrade

nurikimchi committed:
Add field trip sections

Added Stanford, Schilling Robotics, Computer History Museum

nurikimchi committed:
Merge pull request #30 from nurikimchi/nuri

Add field trip sections

nurikimchi committed:
Add testimonial side-by-side section

Remove old testimonial

nurikimchi committed:
Merge pull request #31 from nurikimchi/nuri

Add testimonial side-by-side section

nurikimchi committed:
Add Field Trip section to mobile navbar; update project version number

nurikimchi committed:
Update ECS preview, update testimonials

nurikimchi committed:
Merge pull request #32 from nurikimchi/nuri

Update ECS preview

nurikimchi committed:
Update learning pathway outcomes preview on course card under Pathway Courses

nurikimchi committed:
Merge pull request #35 from nurikimchi/nuri

Update learning pathway outcomes preview on course card under Pathway Courses in the main webpage

nurikimchi committed:
Add new alumni institution logos, adjust spelling on mobile navbar

nurikimchi committed:
Merge pull request #36 from nurikimchi/nuri

Add new alumni institution logos, adjust spelling on mobile navbar

TristanCode0327 committed:
Remove registration banner
Remove dates in classes
Edweek Video / article on ML + testimonials
new pics to AP CSP/ CS&RB/ MLH/ SkillsUSA

TristanCode0327 committed:
Add new images and testimonials for various courses; update opportunities data

- Added multiple new images for CSFHS, CS Robotics, ML Honors, and Skills USA opportunities.
- Removed outdated images from Skills USA section.
- Added new testimonials for Machine Learning course highlighting student experiences.
- Updated opportunities data to reflect recent achievements.
- Removed year display from various pages for a cleaner look.
- Introduced EducationWeek logo and link in the ML Honors course page.

TristanCode0327 committed:
Add Zoo Project content and fix dropdown prop bindings

nurikimchi committed:
Fix ImageCarousel.svelte import error
